src/cmd/package.go

```diff
@@ -195,29 +195,23 @@ var packageRemoveCmd = &cobra.Command{
 		defer pkgClient.ClearTempPaths()
 
 		if err := pkgClient.Remove(pkgName); err != nil {
-			message.Fatalf(err, "Unable to remove the package with an error of: %s", err.Error())
+			message.Fatalf(err, lang.CmdPackageRemoveErr, err.Error())
 		}
 	},
 }
 
 var packagePublishCmd = &cobra.Command{
-	Use:   "publish [PACKAGE|SKELETON DIRECTORY] [REPOSITORY]",
-	Short: "Publish a Zarf package to a remote registry",
-	Example: `
-# Publish a package to a remote registry
-zarf package publish my-package.tar oci://my-registry.com/my-namespace
-
-# Publish a skeleton package to a remote registry
-zarf package publish ./path/to/dir oci://my-registry.com/my-namespace
-`,
-	Args: cobra.ExactArgs(2),
+	Use:     "publish [PACKAGE|SKELETON DIRECTORY] [REPOSITORY]",
```

Comment:
Just noting that `Use:` strings are the only keys that are still using magic strings instead of a declared variable string. Was this on purpose?

Reply:
Yes - IMO it is useful to visually index off of them and (at least from what I have seen) a lot of docs don't change them because the commands and flags themselves will be the same in other languages: https://wiki.debian.org/fr/AptCLI

Reply:
Though that said some things are translated so it can be context specific (i.e. `<nom-paquet>`)